schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/util/KeyUtil.java

```diff
@@ -50,10 +51,11 @@
             .put(ApplicationMeta.get().getKind(), "a") //App
             .put(ApplicationPropertyMeta.get().getKind(), "ap") //AppProp
             .put(AttachmentMeta.get().getKind(), "ATT")
+            .put(ContactMessageMeta.get().getKind(), "CM") //ContactMessage
```

Comment:
I bet you the only reason you knew this was needed is because a test failed, oder?
I got caught twice before writing that test :-)